### ISSUE TITLE
[TextDisplayMenu] Cannot have sub-menu of 1 item

### ISSUE THREAD

**Issue by patridge** [WildernessLabs/Meadow.Foundation#381]
If you have a menu with a single sub-item, it will not be recognized as a sub-menu. Selecting the resulting menu item it will neither trigger `Selected` nor open a sub-menu.

```csharp
var menuItems = new MenuItem[]
{
    ...
    new MenuItem("Options", subItems: new MenuItem[]
    {
        new MenuItem($"Version {version}"),
    }),
};
```

**Comment by patridge** [comment]
This appears to be related to the `HasSubItems` property looking for `SubItems.Length > 1`.

I don't know the other implications of this change, but it definitely resolves the issue for me when I change that to zero.

I put together PR #382 that fixes the issue for me, if we aren't concerned about other menu interactions that won't like single-item menus.

**Comment by patridge** [comment]
Wanted to fix in `develop` but accidentally targeted `main` with the PR. This is fixed by #384 in develop.
